azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/OSProfile.java

```diff
@@ -91,15 +90,52 @@ public static LinuxConfiguration create(final String disablePasswordAuthenticati
 
       @AutoValue
       public abstract static class WinRM {
+          public enum Protocol {
+
+              HTTP("http"),
+              HTTPS("https"),
+              UNRECOGNIZED("Unrecognized");
```

Comment:
@bostko I think it's better to produce an UNRECOGNIZED value rather than an exception or null, if we get back some different value. For example, if microsoft add a third possible value in their API, which jclouds doesn't understand, then we'll say that is `UNRECOGNIZED`.